docs/best-practices/feature-flag-cleanup.md

```diff
@@ -31,13 +31,23 @@ Practicing healthy feature flag hygiene is a good way to mitigate technical debt
 
 You should also remove a flag or its variables if they are inactive or not in use. Perhaps the feature was never shipped, or it was shipped with a different flag or variable. Either way, removing it will clear the clutter in your dashboard and your code.
 
+## Using DevCycle CLI to cleanup feature flags
+
+Currently only supported in JavaScript.
+
+Our CLI provides the functionality of removing all instances of a feature flag automatically. 
+
+The `dvc cleanup` command scans your code for all instances of a DevCycle Variable throught the provided key, all references are changed to the value provided removing all `getVariable` or `getVariableValue` calls along with simplifying any `if` statement and blocks with the desired value. 
```

Comment:
`getVariable` or `getVariableValue` might be confusing, since the method names are actually `variable`. `useVariable` and `useVariableValue`

Reply:
also typo? "throught"